src/.nuget/Microsoft.NETCore.Jit/descriptions.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "Name": "RuntimePackage",
+        "Description": "Internal implementation package not meant for direct consumption.  Please do not reference directly.",
+        "CommonTypes": [ ]
+    },
+    {
+        "Name": "Microsoft.NETCore.Jit",
```

Comment:
I agree that we should be careful about encoding the name of the native binary in the package ID, especially if that might change overtime or have a different name on different platforms, or in some cases might carry more than one binary.